src/main/webapp/navbar.html

```diff
@@ -0,0 +1,49 @@
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark mb-5">
+  <div class="container">
+    <a class="navbar-brand" href="#">Entertainment Hub</a>
+    <button
+      class="navbar-toggler"
+      type="button"
+      data-toggle="collapse"
+      data-target="#navbarSupportedContent"
+      aria-controls="navbarSupportedContent"
+      aria-expanded="false"
+      aria-label="Toggle navigation"
+    >
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <div class="form-inline my-2 my-lg-0">
+        <input
+          class="form-control mr-sm-2"
+          type="search"
+          name="searchValue"
+          id="searchValue"
+          placeholder="Search by Title"
+          oninput="getDashboardItems()"
+        />
+        <select
+          class="form-control my-2 my-sm-0 mr-sm-2"
+          name="sortingDirection"
+          id="sortingDirection"
+          required
+        >
+          <option value="ASCENDING" selected
+            >Title (Increasing Alphabetical Order)</option
+          >
+          <option value="DESCENDING"
+            >Title (Decreasing Alphabetical Order)</option
+          >
+        </select>
+        <button
+          type="button"
+          class="btn btn-light navbar-btn text-nowrap my-sm-0"
+          data-toggle="modal"
+          data-target="#itemSubmissionDialog"
+        >
+          Add Entertainment Item
+        </button>
```

Comment:
Not a huge deal, but since the more common case is to not show these items in the nav bar, maybe navbar.html has just the basic navbar and you have another html file that you load in these items

Reply:
Yeah I am going to make it so the search and sorting features of the navbar are hidden by default in the next PR.